js_modules/dagster-ui/packages/ui-core/src/graphql/types.ts

```diff
@@ -3048,6 +3048,7 @@ export type PartitionBackfill = {
   id: Scalars['String']['output'];
   isAssetBackfill: Scalars['Boolean']['output'];
   isValidSerialization: Scalars['Boolean']['output'];
+  logEvents: InstigationEventConnection;
```

Comment:
This looks great, but I wonder if there are also changes to a `schema.graphql` file that need to be committed here?  I think that Typescript may get mad about the changes in this file without them being present there too.

To be sure, you could run `make generate:graphql` in the `js_modules/dagster-ui` folder on your branch.

Reply:
yeah i think so too. i'm having issues running the make command though, looking into it

Reply:
done!